lib/core/types.hpp

```diff
@@ -7,6 +7,10 @@ enum class DigitalSignal { kLow = 0, kHigh };
 
 using Float = float;
 
+#if LINUX
+#include <linux/can.h>
+struct CanFrame = can_frame;
```

Comment:
Is this the best way to do this? I'm not confident with preprocessor directives but it seems weird to have to include `<linux/can.h>` here

Reply:
Actually this straight up doesn't compile, but I'm not sure how to do it right